.coveragerc

```diff
@@ -3,6 +3,8 @@ branch = True
 data_file = .coverage
 source=commerce_coordinator
 omit =
+    */tests/*
+    commerce_coordinator/apps/demo_lms/*
```

Comment:
- We should include tests in our coverage! [[why](https://nedbatchelder.com/blog/202008/you_should_include_your_tests_in_coverage.html)]
- Is there a reason to exclude `demo_lms`?

Reply:
Just trying to drop the coverage number... and demo_lms really should just be removed eventually... were not building anything new there...

Reply:
As far as tests, they were running so i dont know why it was saying they werent covered... (i also dont entirely agree with the article, but im happy to uncomment them...)